pkg/cache/json.go

```diff
@@ -58,31 +60,66 @@ func (q *JSON) ListBundles(ctx context.Context) ([]*api.Bundle, error) {
 }
 
 func (q *JSON) SendBundles(_ context.Context, s registry.BundleSender) error {
+	var keys []apiBundleKey
 	for _, pkg := range q.packageIndex {
-		channels := sets.KeySet(pkg.Channels)
-		for _, chName := range sets.List(channels) {
-			ch := pkg.Channels[chName]
-
-			bundles := sets.KeySet(ch.Bundles)
-			for _, bName := range sets.List(bundles) {
-				b := ch.Bundles[bName]
-				apiBundle, err := q.loadAPIBundle(apiBundleKey{pkg.Name, ch.Name, b.Name})
-				if err != nil {
-					return fmt.Errorf("convert bundle %q: %v", b.Name, err)
-				}
-				if apiBundle.BundlePath != "" {
-					// The SQLite-based server
-					// configures its querier to
-					// omit these fields when
-					// bundle path is set.
-					apiBundle.CsvJson = ""
-					apiBundle.Object = nil
-				}
-				if err := s.Send(apiBundle); err != nil {
-					return err
-				}
+		for _, ch := range pkg.Channels {
+			for _, b := range ch.Bundles {
+				keys = append(keys, apiBundleKey{pkg.Name, ch.Name, b.Name})
+			}
+		}
+	}
+	sort.Slice(keys, func(i, j int) bool {
+		if keys[i].chName != keys[j].chName {
+			return keys[i].chName < keys[j].chName
+		}
+		if keys[i].pkgName != keys[j].pkgName {
+			return keys[i].pkgName < keys[j].pkgName
+		}
+		return keys[i].name < keys[j].name
+	})
```

Comment:
Why are we ordering the list here?

Reply:
For e.g. https://github.com/operator-framework/operator-registry/issues/1069